ci/scripts/jenkins_build.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+source ci/scripts/jenkins_common.sh
+
+gpuci_logger "Creating conda env"
+conda config --add pkgs_dirs /opt/conda/pkgs
+conda config --env --add channels conda-forge
+conda config --env --set channel_alias ${CONDA_CHANNEL_ALIAS:-"https://conda.anaconda.org"}
+mamba install -q -y -n base -c conda-forge "boa >=0.10"
+mamba create -q -y -n morpheus python=${PYTHON_VER}
+conda activate morpheus
+
+gpuci_logger "Installing CI dependencies"
+mamba env update -q -n morpheus -f ./docker/conda/environments/cuda${CUDA_VER}_ci.yml
```

Comment:
Lets make everything relative to `MORPHEUS_ROOT`
```suggestion
mamba env update -q -n morpheus -f ${MORPHEUS_ROOT}/docker/conda/environments/cuda${CUDA_VER}_ci.yml
```